src/main/java/lotto/LottoTickets.java

```diff
@@ -0,0 +1,28 @@
+package lotto;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class LottoTickets {
+    private final List<LottoTicket> lottoTickets;
+
+    public LottoTickets(List<LottoTicket> lottoTickets){
+        this.lottoTickets = lottoTickets;
+    }
+
+    public static LottoTickets of(int ticketCount){
+        List<LottoTicket> tickets = new ArrayList<>();
+        for(int i=0;i<ticketCount;i++){
```

Comment:
```suggestion
        for(int i = 0; i < ticketCount; i++){
```
인텔리 제이등의 ide 에서 제공하는 reformat 기능을 이용하시면 자바의 범용적인 컨벤션을 쉽고 한번에 적용하실 수 있습니다 :)